### DISCUSSION TITLE
Group Column in Excel using abap2xlsx

### DISCUSSION THREAD

**Discussion by ShamoMikayil** [abap2xlsx/abap2xlsx#1181] [General]
Hi Experts,

short question:: Is it possible to group the columns in Excel using abap2xlsx?
I need just group column (under section Data in Excel) before exporting it.

Thanks in advance

Shamo

**Comment by ShamoMikayil** [comment]
Update: after update it works fine :)

**Comment by sandraros** [comment]
Outline levels are demonstrated in the program [`zdemo_excel12`](https://github.com/abap2xlsx/demos/blob/main/src/zdemo_excel12.prog.abap):

https://github.com/abap2xlsx/demos/blob/d3b82106f41cf5bd04a8967d62ab33585e96ccd7/src/zdemo_excel12.prog.abap#L67-L72

![image](https://github.com/abap2xlsx/abap2xlsx/assets/34005250/2312c359-2bfb-4a6c-83d3-0b9d80602064)

**Comment by ShamoMikayil** [comment]
Thanks Sandaros !
